bin/publish.ml

```diff
@@ -23,7 +23,9 @@ let publish_doc ~dry_run ~yes pkg_names pkg =
   >>= fun dir -> OS.Dir.exists dir
   >>= fun force -> Pkg.infer_pkg_names dir pkg_names
   >>= fun pkg_names ->
-  App_log.status (fun l -> l "Selected packages: %a" Fmt.(list (styled `Bold string)) pkg_names);
+  App_log.status (fun l -> l "Selected packages: %a"
+                     Fmt.(list ~sep:(unit "@,") (styled `Bold string))
```

Comment:
I think it should be `"@ "` and not `"@,"`, see `print_cut` documentation in `Format`: https://caml.inria.fr/pub/docs/manual-ocaml/libref/Format.html